examples/image_classification-tf.ipynb

```diff
@@ -0,0 +1,7883 @@
+{
```

Comment:
<p>Should we also mention ViT MAE since it differs from the rest of the models its pre-training strategy (it's a self-supervised model)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I don't think ViT MAE has a classification head? As we're using <code>TFAutoModelForImageClassification</code>, we can only load models specified in the <code>TF_MODEL_FOR_IMAGE_CLASSIFICATION_MAPPING</code>   </p>

Reply:
ViTMAE directly fits into `ViTForImageClassification`.

Reply:
<p>As I understand it, we can load VitMAE weights into a ViT model with <code>model = TFVitForImageClassification.from_pretrained(vit_mae_checkpoint)</code>  but there isn't a <code>TFVitMAEForImageClassification</code> object and <code>TFAutoModelForImageClassification.from_pretrained(vit_mae_checkpoint)</code>  throws an error because the model type isn't specified in <code>TF_MODEL_FOR_IMAGE_CLASSIFICATION_MAPPING</code>   (at least when I try). As a VitMAE checkpoint can't be used with <code>TFAutoModelForImageClassification</code> I didn't want to cause confusion and include it. </p><p> </p><p>I can mention and add a clarifying note?</p>